arviz/plots/bpvplot.py

```diff
@@ -44,50 +44,50 @@ def plot_bpv(
+           * The ``kind="u_value"`` argument computes :math:`p_i := p(y_i* \leq y_i | y)`. i.e. like a p_value 
+             but per observation :math:`y_i`. This is also known as marginal p_value. The ideal distribution 
+             is uniform. This is similar to the LOO-pit calculation/plot, the difference is than in 
+             LOO-pit plot we compute :math:`pi = p(y_i* r \leq y_i | y_{-i} )`, where :math:`y_{-i}`, is all 
+             other data except :math:`y_i`.
+           * The ``kind="t_stat"`` argument computes :math:`:= p(T(y)* \leq T(y) | y)` where T is any test statistic. 
+             See ``t_stat`` argument below for details of available options.
+             
+    t_stat : {"mean", "median", "std"},  default "median"
+        Test statistics to compute from the observations and predictive distributions. Alternative a quantile 
+        can be passed as a float (or str) in the interval (0, 1). Finally a user defined function is also
         acepted, see examples section for details.
     bpv : bool, default True
         If True add the Bayesian p_value to the legend when ``kind = t_stat``.
     plot_mean : bool, default True
         Whether or not to plot the mean test statistic.
-    reference : str, default "analytical"
-        How to compute the distributions used as reference for u_values or p_values. Allowed values
-        are "analytical" and "samples". Use `None` to do not plot any reference.
-        Defaults to "samples".
+    reference : {None,"analytical","samples"}, default "analytical"
+        How to compute the distributions used as reference for ``kind=u_values`` or ``kind=p_values``. 
+        Use `None` to do not plot any reference.
     mse : bool, default False
         Show scaled mean square error between uniform distribution and marginal p_value
         distribution.
     n_ref : int, default 100
         Number of reference distributions to sample when ``reference=samples``.
     hdi_prob : float, optional
         Probability for the highest density interval for the analytical reference distribution when
-        computing u_values. Should be in the interval (0, 1]. Defaults to the
+        ``kind=u_values``. Should be in the interval (0, 1]. Defaults to the
         rcParam ``stats.hdi_prob``.
     color : str, optional
         Matplotlib color
     grid : tuple, optional
-        Number of rows and columns. Defaults to None, the rows and columns are
+        Number of rows and columns. By default, the rows and columns are
         automatically inferred.
-    figsize : (float, float), optional
+    figsize : tuple, optional
         Figure size. If None it will be defined automatically.
     textsize : float, optional
-        Text size scaling factor for labels, titles and lines. If None it will be
-        autoscaled based on ``figsize``.
+        Text size scaling factor for labels, titles and lines. If None it will be autoscaled based
+        on ``figsize``.
```

Comment:
it was already wrong before, but here figsize should have only single backticks as it is referencing a parameter alone.

Reply:
ok